### PR TITLE
Align actions-prepend slot items to the left on v-data-table 

### PR DESCRIPTION
**Pull request by mrodal**
## Description

Added flex:1 to prepend-actions div on v-data-table and data-iterator

## Motivation and Context

Fixes https://github.com/vuetifyjs/vuetify/issues/5477

## How Has This Been Tested?

visually

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [X] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.

### CODE REVIEW

File: packages/vuetify/src/stylus/components/_data-iterator.styl [hunk 1]
```diff
@@ -52,3 +52,6 @@ theme(v-dataiterator, "v-data-iterator")
 
       .v-select__selection--comma
         font-size: 12px
+
+  &__prepend-slot
+    flex: 1
```
Comment (johnleider) [inline packages/vuetify/src/stylus/components/_data-iterator.styl]: This has the potential to cause issues on IE11

**Comment by nekosaur** [comment]
I'm fine with adding a class that could be used to target some custom css. I would probably leave what that css is up to whomever uses the slot. Another solution could also be to skip the wrapping div entirely and just render the slot.

**Comment by mrodal** [comment]
Yeah, I thought about just adding a class and let the developer do whatever he wants with it, but I also thought that it was nice to display the items better that it is now, that doesnt look right.

Skipping the div maybe makes the aligning of the items harder?

**Comment by nekosaur** [comment]
Skipping the div, you could do `<div slot="prepend-slot" class="d-flex">...</div>`

**Comment by mrodal** [comment]
Tru dat, I like it

**Comment by mrodal** [comment]
Ok, now I just add the flex class to the slot div. I didnt know about that class 😁

**Comment by johnleider** [comment]
Waiting for @nekosaur 's final approval.

**Comment by nekosaur** [comment]
If you want to keep the wrapping div, then please add back the `__prepend-slot` class so that it can be easily targeted with css selectors.

**Comment by KaelWD** [comment]
We can't use .flex, it won't work unless the user has loaded it themselves. This should be its own table-specific class and css.

File: packages/vuetify/src/mixins/data-iterable.js [hunk 2]
```diff
@@ -474,7 +474,9 @@ export default {
       return [this.$createElement('div', {
         'class': this.actionsClasses
       }, [
-        this.$slots['actions-prepend'] ? this.$createElement('div', {}, this.$slots['actions-prepend']) : null,
+        this.$slots['actions-prepend'] ? this.$createElement('div', {
+          'class': 'flex'
```
Comment (KaelWD) [inline packages/vuetify/src/mixins/data-iterable.js]: We can't use .flex, it won't work unless the user has loaded it themselves. This should be its own table-specific class and css.

**Comment by johnleider** [comment]
Closing due to inactivity.

**Pull request closed without merging**